### PR TITLE
Make it work with Swift 2 and fix all the warnings.

### PR DESCRIPTION
**Pull request by aranasaurus**

### CODE REVIEW

File: IKAsyncable/IKAsyncCollectionViewDelegate.swift [hunk 1]
```diff
@@ -29,13 +29,12 @@ public class IKAsyncCollectionViewDelegate : IKAsyncOperationManager, UICollecti
...
         super.resetOperations()
         
-        if let collectionView = self.collectionView,
-            let indexPaths = collectionView.indexPathsForVisibleItems() as? [NSIndexPath] {
-                for indexPath in indexPaths  {
-                    if let cell = collectionView.cellForItemAtIndexPath(indexPath) {
-                        self.handleCell(cell, indexPath: indexPath)
-                    }
+        if let collectionView = self.collectionView {
+            for indexPath in collectionView.indexPathsForVisibleItems() {
```
Comment (IanKeen) [inline IKAsyncable/IKAsyncCollectionViewDelegate.swift:33]: oooh is this casted correctly now?

Comment (aranasaurus) [inline IKAsyncable/IKAsyncCollectionViewDelegate.swift:33]: Yup most NSArrays are, maybe even all ;)

Comment (IanKeen) [inline IKAsyncable/IKAsyncCollectionViewDelegate.swift:33]: ahh so nice!

**Comment by IanKeen** [comment]
Can you please check that the Swift-2.0 branch is ok; If so we can get rid of this PR

**Comment by aranasaurus** [comment]
New PR incoming!

**Pull request closed without merging**